widget/layouts.go

```diff
@@ -0,0 +1,60 @@
+package widget
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+)
+
+// Layout of multiple widgets
+// Each layout is stored serverside to be retrieved later by the client
+type Layout struct {
+	Name    string
+	Widgets []*Widget
+}
+
+func getFilename(name string) string {
+	return fmt.Sprintf("layouts/%s.json", name)
```

Comment:
I think this is assuming a certain file structure? In docker, we are running a single executable without any folders such as `layouts`. I think we should make sure that the folder exists and create it if it does not, since that would be the case once deployed.

Reply:
Yeah I remember now we ran into this issue with Tim. I can add that check